### PR TITLE
Updated linter line length to something less draconian.

### PR DESCRIPTION
**Pull request by blackbaud-brandonhare**

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -82,7 +82,7 @@
     ],
     "max-line-length": [
       true,
-      100
+      200
```
Comment (Blackbaud-SteveBrush) [inline tslint.json]: I'm going to disagree here, but will defer to @Blackbaud-BobbyEarl. The line length might seem too short on a widescreen monitor, but shorter lines are easier to read and enforce fewer tab structures.

Obviously this is opinion based, but I think the consensus is 80-100 characters.

Interesting read: https://stackoverflow.com/questions/110928/is-there-a-valid-reason-for-enforcing-a-maximum-width-of-80-characters-in-a-code

Comment (blackbaud-brandonhare) [reply tslint.json]: I understand the rationale, but for me personally, it is really obnoxious with strings. I don't think I should have to modify my string literals to be concatinated or array.joined to pass a linter rule. There has to be some kind of happy medium between the two.

Comment (blackbaud-brandonhare) [reply tslint.json]: I could also pepper my code with `//Ignore rule`, but that seems just as bad.

Comment (Blackbaud-SteveBrush) [reply tslint.json]:
```
const str = [
  'a',
  'long',
  'string'
].join('');
```
^ To me, that's a thing of beauty :)

Comment (blackbaud-brandonstirnaman) [reply tslint.json]: I'm a huge proponent of longer lines than we allow today. I generally fit 120-140 characters per line without scrolling on my laptop.

**Comment by Blackbaud-LuisBello** [comment]
i endorse this pr :us:

**Pull request merged**